File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.java
class Solution {
    public int minElement(int[] nums) {
        int min=nums[0];
        for (int i=0;i<nums.length;i++){
            int sum=0;
            while (nums[i]!=0){
                int a=nums[i]%10;
                sum=sum+a;
                nums[i]=nums[i]/10;
            }
            if(sum<=min){
                min=sum;
            }
        }
        return min;
    }
}

File: 3300-minimum-element-after-replacement-with-digit-sum/3300-minimum-element-after-replacement-with-digit-sum.py
class Solution(object):
    def minElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        min = nums[0]

        for i in range(len(nums)):
            sum = 0

            while nums[i] != 0:
                a = nums[i] % 10
                sum = sum + a
                nums[i] = nums[i] // 10

            if sum <= min:
                min = sum

        return min